mbed/mbed.py

```diff
@@ -2759,6 +2778,78 @@ def toolchain_(name=None, global_cfg=False, supported=False):
         return compile_(supported=supported)
     return config_('toolchain', name, global_cfg=global_cfg)
 
+
+@subcommand('cache',
+    dict(name='on', nargs='?', help='Turn repository caching on. Will use either the default or the user specified cache directory.'),
+    dict(name='off', nargs='?', help='Turn repository caching off. Note that this doesn\'t purge cached repositories. See "purge".'),
+    dict(name='dir', nargs='?', help='Set cache directory. Set to "default" to let mbed CLI determine the cache directory location. Typically this is "~/.mbed/mbed-cache/" on UNIX, or "%%userprofile%%/.mbed/mbed-cache/" on Windows.'),
```

Comment:
Is it possible to actually print out the folder it WOULD really use rather than guess? Do we know at this point or can we use just %s and substitute it in?

Reply:
Is this what you mean?

```
$ mbed cache
[mbed] Repository cache is ENABLED.
[mbed] Cache location "/Users/mihsto01/.mbed"
```

Note that the actual folder is /Users/mihsto01/.mbed/mbed-cache

Reply:
I think Janne is asking about the help. so `mbed cache -h` prints something OS + user specific.

Reply:
Pushed minor patch to support this.